lib/matplotlib/backends/web_backend/js/nbagg_mpl.js

```diff
@@ -7,6 +7,17 @@ var comm_websocket_adapter = function (comm) {
     var ws = {};
 
     ws.binaryType = comm.kernel.ws.binaryType;
+    ws.readyState = comm.kernel.ws.readyState;
```

Comment:
Why does this not need to wrapped in an `if` while the version in the function does? Otherwise this all makes sense to me an :+1:  it does indeed allow resizing.

Reply:
ooh I think I see. The version in the function needs to not error out even after the kernel ws closes?

Reply:
Yes, when closed, it will be `null` and no longer have `readyState` on it.